production/ksonnet/loki/multi-zone.libsonnet

```
@@ -17,29 +17,38 @@
   },
 
   _config+: {
-    multi_zone_ingester_enabled: true,
-    multi_zone_ingester_migration_enabled: false,
-    multi_zone_ingester_replication_write_path_enabled: true,
-    multi_zone_ingester_replication_read_path_enabled: true,
-    multi_zone_ingester_replicas: 0,
-    multi_zone_ingester_max_unavailable: std.max(1, std.floor($._config.multi_zone_ingester_replicas / 9)),
-    multi_zone_default_ingester_zone: false,
-    multi_zone_ingester_exclude_default: false,
+    multi_zone_ingesters: {
+      enabled: true,
+      migration_enabled: false,
+      replication_write_path_enabled: true,
+      replication_read_path_enabled: true,
+      num_replicas: 0,
+      max_unavailable: std.max(1, std.floor($._config.multi_zone_ingester.num_replicas / 9)),
```

I think this should be:

```suggestion
      max_unavailable: std.max(1, std.floor($._config.multi_zone.num_replicas / 9)),
```